### ISSUE TITLE
♿Data Table AVT1 - August 2018♿

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon#1342]
## Environment
Chrome Version 70.0.3538.67 (Official Build) (64-bit)
macOS Mojave 10/Windows 10

### General Table:
The search bar's icons as well as the down/up arrow for sorting the table are not visible in Windows High Contrast Mode
![capture](https://user-images.githubusercontent.com/40970507/47531165-6a887800-d872-11e8-807d-55a36f0e5234.PNG)

Provide descriptive text in label elements
![capture](https://user-images.githubusercontent.com/40970507/47460969-4743c700-d7a5-11e8-9374-d05487a6f25c.png)

Each form control must have associated label
![capture](https://user-images.githubusercontent.com/40970507/47460870-077cdf80-d7a5-11e8-8506-f03b0f576ea8.png)

The WAI-ARIA role search is incorrectly applied to the element input. A form can have a role of search.
![capture](https://aat.w3ibm.mybluemix.net/token/b62ec93e-0a1f-4e82-b96a-18e75818d205/51d217d2-9b4e-4e3e-a04e-66809340d9d0/latest///doc/w3/help/en-US/idhi_accessibility_check_g1146.html)

### Expandable Table
Each form control must have an associated label
![capture](https://user-images.githubusercontent.com/40970507/47511686-575cb480-d840-11e8-9302-335744788b87.png)

### Selectable Table
Provide descriptive label text for an element
![capture](https://user-images.githubusercontent.com/40970507/47513998-04393080-d845-11e8-8c79-1c9bb2b6b303.png)

Search landmarks must have a unique label
![capture](https://user-images.githubusercontent.com/40970507/47514126-45314500-d845-11e8-8dbe-6eb27ee57908.png)

## How to reproduce
[Download the Dynamic Accessibility Plugin tool here
](https://ibm.ent.box.com/notes/335005856253?s=mhpgjlgss354mcidsv6rvpuu0bxlmfm4)Run the extension using the August 2018 ruleset

**Comment by dakahn** [comment]
Closing this -- stale info and we're hitting this in our component audit this quarter.
